addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -1098,6 +1110,22 @@ function normalizeRem($editable, rootFontSize=16) {
         if (td.closest('.s_masonry_block')) {
             outlookTd.style.padding = 0; // Not sure why this is needed.
         }
+        // Outlook doesn't support left/right padding on images. When we can
+        // (when the image is the only child of its parent), apply said padding
+        // to the parent.
```

Comment:
That's nitpicking, but I see no point for "When we can (explanation of when we can)". It just complicates the sentence for no added benefit. I suggest the more direct version, which even holds on two lines now:
```js
        // Outlook doesn't support left/right padding on images. When the
        // image is the only child of its parent, apply said padding to the parent.
```

Reply:
I put it that way to make clear that this is a partial solution since we can only apply it in that case. But I think you're right anyway.

Reply:
Done.